### PR TITLE
Adapt to coq/coq#15693

### PR DESCRIPTION
**Pull request by proux01**
To be merged in sync with the upstream PR coq/coq#15693

I had to comment a line in the Makefile for the installation to work properly in the CI of Coq

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,7 +33,7 @@ DESTDIR ?= $(shell $(COQBIN)/coqc -where)/../../
 DOCDIR ?= share/doc/coq-elpi/
 COQDOCINSTALL ?= $(DESTDIR)$(DOCDIR)/user-contrib
 
-export DESTDIR
+#export DESTDIR
```
Comment (CohenCyril) [inline Makefile:36]: Ahah yes, srry, but I think this is not the right fix. I think the split between `DESTDIR` and `DOCDIR` was wrong...
Is it supposed to be
```
DESTDIR ?= $(shell $(COQBIN)/coqc -where)/../
DOCDIR ?= ../share/doc/coq-elpi/
```
maybe?
CC @gares

**Comment by proux01** [comment]
Is there a difference between `ifndef X \n X=V \n endif` and `X?=V`? If not, the latter avoids any risk of mispelling one of the two instances of `X`.

**Comment by gares** [comment]
I don't think there is any. But when we did that change we had an extra `export` and I'm not so sure if exporting an empty variable is the same as not exporting it. Then the code changed and the ifdef stayd :-/

**Comment by ppedrot** [comment]
@gares Coq-elpi still seems to be broken on master after this overlay, see e.g. https://gitlab.com/coq/coq/-/jobs/2406603188.

**Pull request merged**